src/main/java/io/appform/dropwizard/sharding/DBShardingBundleBase.java

```diff
@@ -117,7 +115,7 @@ private void init(final ImmutableList<Class<?>> inEntities) {
         val blacklistingStore = getBlacklistingStore();
         this.shardManager = createShardManager(numShards, blacklistingStore);
         this.shardInfoProvider = new ShardInfoProvider(dbNamespace);
-        this.healthCheckManager = new HealthCheckManager(dbNamespace, shardManager, shardInfoProvider);
+        this.healthCheckManager = new HealthCheckManager(dbNamespace, shardInfoProvider, blacklistingStore, shardManager);
```

Comment:
Move these to constructor and make the properties final if possible

Reply:
This is part of init method called by each constructor. One of the constructor has few preprocessing steps which eliminates constructor chaining

Reply:
Done